openwisp_notifications/tests/test_transaction_notifications.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth import get_user_model
+from django.contrib.contenttypes.models import ContentType
+from django.core.cache import cache
+from django.db.models.signals import post_save
+from django.test import TransactionTestCase
+
+from openwisp_notifications.handlers import register_notification_cache_update
+from openwisp_notifications.signals import notify
+from openwisp_notifications.swapper import load_model
+from openwisp_users.tests.utils import TestOrganizationMixin
+
+User = get_user_model()
+Notification = load_model('Notification')
+
+
+class TestTransactionNotifications(TestOrganizationMixin, TransactionTestCase):
+    def setUp(self):
+        self.admin = self._create_admin()
+        self.notification_options = dict(
+            sender=self.admin,
+            description='Test Notification',
+            level='info',
+            verb='Test Notification',
+            email_subject='Test Email subject',
+            url='https://localhost:8000/admin',
+        )
+
+    def _create_notification(self):
+        return notify.send(**self.notification_options)
+
+    def _register_notification_cache_update(self, model, signal, dispatch_uid):
+        register_notification_cache_update(model, signal, dispatch_uid=dispatch_uid)
```

Comment:
I don't see the use of this method. It is simply calling the other function. Why is this required?